.circleci/config.yml

```diff
@@ -32,7 +32,10 @@ jobs:
       - *cache_restore
       - run:
           name: Release
-          command: sbt ';set version in ThisBuild := "'`git describe`'"; test; publish'
+          command: |
+            set -e
+            tags=`git describe --tags`
```

Comment:
Shouldn't it be `git describe --tags --abbrev=0` instead to exclude the hash and the number of commits since the tag, so instead of e.g. `0.1.0-5-ba7f4f5` you will get just `0.1.0`.

Reply:
when you're on the same commit as a tag it just returns the tag without suffixes, and this job only runs on tagged commits

Reply:
cool, in that case LGTM